lib/matplotlib/tight_layout.py

```diff
@@ -123,6 +123,9 @@ def auto_adjust_subplotpars(fig, renderer,
     for subplots, ax_bbox, (num1, num2) in zip(subplot_list,
                                                ax_bbox_list,
                                                num1num2_list):
+        if all([not ax.get_visible() for ax in subplots]):
```

Comment:
yeah, I don't understand why you changed what I originally suggested. I suggested putting a filter in the list comprehension down below. Now, it does make sense to skip this iteration if *all* the axes in this group are invisible. So, I would put the filter in the list comprehension down below, and also change this one from a `any` to an `all`.